ci-operator/step-registry/hypershift/install/hypershift-install-commands.sh

```diff
@@ -41,6 +41,13 @@ if [ "${CLOUD_PROVIDER}" == "AWS" ]; then
   --external-dns-domain-filter=service.ci.hypershift.devcluster.openshift.com \
   --wait-until-available \
   ${EXTRA_ARGS}
+elif [ "${CLOUD_PROVIDER}" == "Azure" ]; then
+  "${HCP_CLI}" install --hypershift-image="${OPERATOR_IMAGE}" \
```

Comment:
can we please also set up external dns here
--external-dns-provider=azure \
--external-dns-credentials 
--external-dns-domain-filter=